build.gradle

```diff
@@ -151,11 +151,9 @@ pmd {
 
 sonarqube {
   properties {
-    property "sonar.projectName", "Reform :: spring-boot-template"
-    property "sonar.projectKey", "uk.gov.hmcts.reform:spring-boot-template"
-    property "sonar.jacoco.reportPath", "${project.buildDir}/jacoco/test.exec"
-    property "sonar.jacoco.itReportPath", "${project.buildDir}/jacoco/integration.exec"
-
+    property "sonar.projectName", "Reform :: RPE PDF Service"
+    property "sonar.projectKey", "uk.gov.hmcts.reform:rpe-pdf-service"
+    property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/jacoco/jacoco.xml"
```

Comment:
does this have the results for unit and integration tests in it?

Reply:
yes it is merged